tests/unit/flow/test_flow_visualization.py

```diff
@@ -17,3 +17,5 @@ def test_visualization():
     flow_created = Image.open(os.path.join(cur_dir, 'flow.jpg'))
     assert flow_original.size == flow_created.size
     np.testing.assert_array_almost_equal(flow_original, flow_created)
+    flow_created.close()
```

Comment:
for proper closing, use `with` context management

Reply:
Got it